### DISCUSSION TITLE
Using DaisyUI with Kobalte

### DISCUSSION THREAD

**Discussion by robodna** [saadeghi/daisyui#3044] [Q&A] [answered]
Would there be a potential issue with using DaisyUI as inputs for Kabolte?

Thank you

**Comment by saadeghi** [comment, marked as answer]
It should work. According to Kobalte docs, their components have no styles by default and you can add class names to them. So it sounds like a good match with daisyUI, since daisyUI is just CSS class names only for styles.

Let me know if you have a question.

**Reply by robodna** [reply]
Thank you!
